project/project-info.conf

```diff
@@ -0,0 +1,46 @@
+project-info {
+  version: "current"
+  shared-info {
+    jdk-versions: ["AdoptOpenJDK 8", "AdoptOpenJDK 11"]
+//    snapshots: {
+//      url: "other-docs/snapshots.html"
+//      text: "Snapshots are available"
+//      new-tab: false
+//    }
+    issues: {
+      url: "https://github.com/akka/akka-projection/issues"
+      text: "GitHub issues"
+    }
+    release-notes: {
+      url: "https://github.com/akka/akka-projection/releases"
+      text: "GitHub releases"
+    }
+    api-docs: [
+      {
+        url: "https://doc.akka.io/api/akka-projection/"${project-info.version}"/akka/projection/"
+        text: "API (Scaladoc)"
+      }
+    ]
+    forums: [
+      {
+        text: "Lightbend Discuss"
+        url: "https://discuss.lightbend.com/c/akka/"
+      }
+      {
+        text: "akka/akka Gitter channel"
+        url: "https://gitter.im/akka/akka"
+      }
+    ]
+  }
+  core: ${project-info.shared-info} {
+    title: "Akka Projection"
+    jpms-name: "akka.projection.core"
+    levels: [
+      {
+        readiness: CommunityDriven
```

Comment:
Should this be `Incubating` (https://github.com/lightbend/lightbend-technology-intro-doc/blob/master/docs/modules/getting-help/pages/support-terminology.adoc) instead?

Reply:
That's a nuance. Maybe when it makes it to a release ;-)